src/open_petro_elastic/material/fluid.py

```diff
@@ -56,12 +68,16 @@ def brie_fluid_mixing(fluids, fluid_saturations, gas, exponent=3.0):
         Engineers, 2018.
     """
     liquid = wood_fluid_mixing(fluids, fluid_saturations)
+    if len(gases) == 0:
```

Comment:
I was wondering if its more clear what is going on if one does:
```python
if len(fluids) != 0:
    liquid = wood_fluid_mixing(fluids, fluid_saturations)
if len(gases) != 0:
    gas = wood_fluid_mixing(gases, gas_saturations)

if len(gases) == 0 and len(liquids) == 0:
    raise ValueError("mixing is undefined for zero constituents")
elif len(gases) == 0:
    return liquid
elif len(liquids) == 0:
    return gas
else:
    total_fluid = sum(fluid_saturations)
    bulk_mix = (liquid.bulk_modulus - gas.bulk_modulus) * (
        total_fluid
    ) ** exponent + gas.bulk_modulus
    density_mix = (
        mix_densities(fluids, fluid_saturations) + (1 - total_fluid) * gas.density
    )
    return fluid_material(density=density_mix, bulk_modulus=bulk_mix)
```

Furthermore, I now see that this function is missing some parameter checking:

```python
if len(fluids) != len(fluid_saturations):
   raise ValueError("Mismatched lengths of fluid and fluid saturation input")
```

Reply:
I think this looks reasonable. It also creates a sort of symmetry between gases and liquids (apart from the mixing equation itself).

`wood_fluid_mixing` will fail if any of the saturations are 0 (division-by-zero error when calculating bulk modulus). This has not been an issue before, since the "gas" bulk modulus and density was not calculated via `wood_fluid_mixing`. This can be fixed by a better 0-saturation handling in `mix_bulk_moduli`, but this seems to require some tests to be updated. I'll submit an update based on this soon.

Reply:
Implemented in the latest update. The saturation issue was instead handled via a `sum_saturations_to_one` function.